packages/rum/src/rum.ts

```diff
@@ -204,15 +199,12 @@ export function startRum(
       lifeCycle.notify(LifeCycleEventType.ACTION_COMPLETED, { context, name, type: UserActionType.CUSTOM })
     }),
     getInternalContext: monitor(
-      (): InternalContext => {
-        return {
-          application_id: applicationId,
-          session_id: viewContext.sessionId,
-          user_action: getUserActionReference(),
-          view: {
-            id: viewContext.id,
-          },
-        }
+      (startTime?: number): InternalContext => {
+        return (withSnakeCaseKeys(deepMerge(
+          { applicationId },
+          parentContexts.findView(startTime),
```

Comment:
`findView` has type `ViewContext | undefined`, but it is required in the `InternalContext` type. Maybe change the `InternalContext view` prop to optional.